tokens/tokens.breakpoints.stories.mdx

```diff
@@ -0,0 +1,55 @@
+import { Meta } from '@storybook/addon-docs'
+
+<Meta
+  title="Design Tokens/Breakpoints"
+  parameters={{
+    viewMode: 'docs',
+    previewTabs: {
+      canvas: {
+        hidden: true
+      }
+    },
+  }} />
+
+# Breakpoints
+
+The following breakpoints are available:
+
+```
+$dp-breakpoints-xl
+$dp-breakpoints-lg
+$dp-breakpoints-md
+$dp-breakpoints-sm
+```
+
+## How to use Breakpoint tokens
+
+### Scss
+
+Breakpoint tokens can be consumed as Scss variables. To use them, they have to be imported at first:
+
+```
+@import '@demos-europe/demosplan-ui/tokens/scss/breakpoints';
+```
+
+Then, tokens can be used within Scss code:
+
+```
+@media all and (min-width: $dp-breakpoints-lg) {
+  .some-selector {
+    display: block;
+  }
+}
+```
+
+
+### Tailwind utilities
+
+As Tailwind is configured with these breakpoints, too, they can be used as modifiers for tailwind utility classes.
+They replace the z-index classes mentioned within the Tailwind documentation.
```

Comment:
Should this be here?

Reply:
no. 93381674f64406a54ce690c36fbfa68141c8f0e3